content/applications/websites/ecommerce.rst

```diff
@@ -28,3 +28,4 @@ products and increase your average cart sizes.
    ecommerce/taxes
    ecommerce/shopper_experience
    ecommerce/maximizing_revenue
+   ecommerce/products
```

Comment:
This will change in one mega PR for a complete doc structure which you can find here: https://docs.google.com/spreadsheets/d/1O7Jg4T8kPE0HaLw-3YDjaj54MMk957-9akBFaSNKQA0/edit#gid=0